README.md

```diff
@@ -34,21 +34,35 @@ public class MyViewModel : ViewModelBase
   // to your IViewModel. It's initial value will be default(int).
   public int MyProperty => this.Get<int>();
 
-  // This will create a DynamicProperty with a name of "MyPropertyFromTask" and an initial value of 10.
+  // This will create and attach a new IDynamicProperty named "MyStringProperty"
+  // to your IViewModel. It's initial value will be "hello". It can be bound two-way.
+  public string MyStringProperty
+  {
+    get => this.Get(initialValue: "hello");
+    set => this.Set(value);
+  }
+
+  // This will create a DynamicProperty with a name of "MyAsyncProperty" and an initial value of 10.
```

Comment:
"and an initial value of 20."

Reply:
Actually the initial value is `default(int)` in this case. I'll update the doc!

Reply:
https://github.com/nventive/Chinook.DynamicMvvm/pull/13